pyanaconda/anaconda.py

```diff
@@ -31,9 +31,11 @@
 from pyanaconda import addons
 
 import logging
-log = logging.getLogger("anaconda")
 stdoutLog = logging.getLogger("anaconda.stdout")
 
+from pyanaconda.ilog import get_logger
```

Comment:
It's all due to naming collisions as far as I can tell (the i likely means Installer):

- ilog - we mostly call the logger we use "log", so a module name should not collide with it
- ihelp - there is built-in Python function called "help" and we should not shadow that
- isys - there is the standard library "sys" module and isys was probably originally meant as an installer specific extension of the sys module
- iutil - could be that there used to be a util module somewhere in the path so iutil was used to avoid naming collisions
- isignal - there is a module called signal in the Python standard library

So it's basically a sort of a custom - add in in front of stuff that would shadow existing modules and built-in functions. But that does not necessarily mean it's correct and could not be improved (using a different name that does not collide, using installer_ prefix instead of just i, etc.).

Still I think that should probably be (with the possible exception of the ilog module) a separate pull request as we already change quite a bit in this one.